src/CommunityToolkit.Maui/Views/DrawingView/DrawingView.shared.cs

```diff
@@ -12,6 +12,20 @@ public class DrawingView : View, IDrawingView
 {
 	readonly WeakEventManager drawingLineCompletedEventManager = new();
 
+	/// <summary>
+	/// Initializes a new instance of the <see cref="DrawingView"/> class.
+	/// </summary>
+	public DrawingView()
+	{
+		Unloaded += DrawingView_Unloaded;
+	}
+
+	void DrawingView_Unloaded(object? sender, EventArgs e)
+	{
+		Unloaded -= DrawingView_Unloaded;
+		Handler?.DisconnectHandler();
+	}
```

Comment:
Thanks Vlad! Let's move this method down next to the other private methods in the file.